extensions/wikia/CuratedContent/CuratedContentSpecialController.class.php

```diff
@@ -198,7 +198,7 @@ public function save() {
 		$this->response->setVal( 'status', $status );
 
 		if ( $status ) {
-			wfRunHooks( 'CuratedContentSave' );
+			wfRunHooks( 'CuratedContentSave', [ $sections ] );
```

Comment:
That function doesn't need to purge section API calls links *yet* → I add VCL caching/purging in #7666. Some adjustments would have to be done there.